src/components/pages/ParentBooking/ParentBookingContainer.js

```diff
@@ -62,8 +62,8 @@ const ParentBookingContainer = props => {
               </Card.Grid>
             </TabPane>
           ))}
-          <TabPane tab="1:1 Sessions" key="3">
-            1:1 Sessions
+          <TabPane tab="My Courses" key="3">
+            My Courses
```

Comment:
Thanks for this change! Are we populating this tab(and other tabs) with our endpoint data or will that be a separate PR?

Reply:
Returning the data in the other tabs will need to be a separate PR.